contracts/epochs/FlowEpoch.cdc

```diff
@@ -606,10 +606,11 @@ pub contract FlowEpoch {
 
         // iterate through all the clusters and create their certificate arrays
         for cluster in clusters {
-            var certificate: FlowEpochClusterQC.ClusterQC = FlowEpochClusterQC.ClusterQC(index: cluster.index, votes: [], voterIDs: [])
+            var certificate: FlowEpochClusterQC.ClusterQC = FlowEpochClusterQC.ClusterQC(index: cluster.index, signatures: [], messages: [], voterIDs: [])
 
             for vote in cluster.votes {
```

Comment:
⚠️  This is including _all_ votes in the output, which will result in an invalid QC if any votes have a different `voteMessage` than the quorum. ⚠️ 

We should only include valid votes in the `ClusterQC` that gets included in the `EpochCommit` event. There are two properties required for a vote to be valid:
1. The `voteSignature` is valid with respect to the `voteMessage` 
2. The `voteMessage` is the same as submitted by a quorum of voters 

We're doing 1 currently, but not fully doing 2. We don't consider a cluster complete until we receive a quorum of votes with consistent `voteMessage`s but we still include votes with inconsistent vote messages in the resulting QC.

Reply:
ok, I didn't realize we only wanted to include valid votes in the QC also. I'll make that change.